README.md

```diff
@@ -100,6 +100,7 @@ See code for all available configurations.
 | [FriendlyARM NanoPC-T4](friendlyarm/nanopc-t4)                      | `<nixos-hardware/friendlyarm/nanopc-t4>`           |
 | [Google Pixelbook](google/pixelbook)                                | `<nixos-hardware/google/pixelbook>`                |
 | [GPD MicroPC](gpd/micropc)                                          | `<nixos-hardware/gpd/micropc>`                     |
+| [Intel NUC 8i7BEH](intel/nuc/8i7beh/)                                     | `<nixos-hardware/intel/nuc/8i7beh>`                |
```

Comment:
Please add also a profile to our flake.nix.

Reply:
Thanks, fixed it. I still use the channels approach, so I forgot to add that up.